ArjunaJTA/jta/tests/classes/com/hp/mwtests/ts/jta/commitmarkable/integration/CMRIntegrationTest.java

```diff
@@ -93,7 +113,8 @@ public void testCMR() throws Exception {
 		doTest(ds);
 	}
 
-	// @Test
+	@Test
+	@Ignore
```

Comment:
Can we activate this test?

Reply:
It was commented out...i.e. I don't know if this can be activated

Reply:
I tested it and it works. But if we don't want it I think we can then remove it. WDYT?

Reply:
I don't know why it was deactivated.

Reply:
It's part of Tom's initial commit (44553931bb3b2a6dbba39434cf6725acf61a3d68) of the feature - ie it has never been activated. The actual test does no asserts and catches any exception so it would never fail.

I think we can safely remove the test but do so in a separate commit.